networking/multiple_networks/configuring-sr-iov.adoc

```diff
@@ -9,39 +9,54 @@ toc::[]
 == About SR-IOV hardware on {product-title}
 
 {product-title} includes the capability to use SR-IOV hardware on your nodes.
-You can attach SR-IOV virtual function (VF) interfaces to Pods on nodes with SR-IOV hardware.
+You can attach SR-IOV virtual function (VF) interfaces to Pods on nodes with
+SR-IOV hardware.
 
 You can use the {product-title} console to install SR-IOV by deploying the
 SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
-components of the SR-IOV stack. The Operator provides following features:
+components of the SR-IOV stack. The Operator provisions following components:
```

Comment:
Minor- the article "the is missing:
"The Operator provisions the following components:"

Reply:
Done